### ISSUE TITLE
CMake Error on devel

### ISSUE THREAD

**Issue by Tobias-Fischer** [robotology/yarp#922]
Hi all,
I get the following cmake error when doing `ccmake ..`:
```
CMake Error at /usr/share/cmake-2.8/Modules/CMakePackageConfigHelpers.cmake:164 (message):
   Unknown keywords given to CONFIGURE_PACKAGE_CONFIG_FILE():
   "INSTALL_PREFIX;/home/tobias/robot/yarp/build"
 Call Stack (most recent call first):
   cmake/YarpDescribe.cmake:48 (configure_package_config_file)
   CMakeLists.txt:86 (include)
```
Is someone already on that?

Best, Tobi

**Comment by drdanz** [comment]
@Tobias-Fischer Should be fixed now, there was a wrong path for CMake < 3.1
Sorry about this...

**Comment by Tobias-Fischer** [comment]
Thanks @drdanz, working nicely.

**Comment by drdanz** [comment]
Fixed on 492ac811fce0861c5bcc5a77064599db35c233f9
